meta.yaml

```diff
@@ -74,6 +74,10 @@ projects:
   type: tool
   slug: query-explorer
   tech: [core_reporting_api, embed_api, management_api, metadata_api]
+- title: Request Composer
+  type: tool
+  slug: request-composer
```

Comment:
You can add a `status` key here so this shows up with a "New!" badge in the sidebar.